File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Set<Integer> numsInSet = new HashSet<>((int)((nums.length/0.75f) + 1));
        for(int num: nums) {
            if(!numsInSet.add(num)){
                return true;
            }
            numsInSet.add(num);
        }
        return false;
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        count = {}
        for num in nums:
            count[num] = 1 + count.get(num, 0)
        
        arr = []
        for num, count in count.items():
            arr.append([count, num])
        
        arr.sort()

        res = []

        while len(res) < k:
            res.append(arr.pop()[1])

        return res
        